assembl/static2/css/components/harvesting.scss

```diff
@@ -0,0 +1,117 @@
+.left-side-harvesting-button {
+  &:hover {
+    .assembl-icon-catch:before {
+      font-weight: bold;
+    }
+
+    .left-side-harvesting-button__label {
+      visibility: visible;
+    }
+
+    .left-side-harvesting-button__button__inside {
+      background-color: #ffed5b;
+    }
+  }
+
+  .left-side-harvesting-button__label {
```

Comment:
When did we start supporting BEM style of naming? The only place in static2 where this style is used is in the searchv1 css files, which was written long before we started implementing control processes on code. @paolina92 @vincentfretin thoughts on the inclusion of BEM style naming for css classes?

https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/

Reply:
I'm ok with BEM to have isolated style at react component level. We should do more of that actually, it allows to avoid style regression on other pages if done well. We can't do BEM all the way because boostrap itself is not BEM. Interesting article: https://evilmartians.com/chronicles/bootstrap-an-intervention
Similarly, at the beginning of the v2, I tried to push styled-components or css modules to have local styles but didn't have much attention.

Reply:
For now from my understanding our rules don't enforce nor forbid the use of any CSS methodology, it's up to the preference of each developer.